README.md

```diff
@@ -39,6 +39,9 @@ We can solve this with expansion:
 
 $$3T\left(\frac{n}{3}\right) + (n)$$
 
+$$4T\left(\frac{n}{4}\right) + (n)$$
+
+
 Thus the time colmplexity of this implementation is $\Theta(n log n)$
```

Comment:
1 - The Plagiarism note is missing. 2 - Why the combination part is O(n)? According to your code, it is not.